cal/cal/static/js/app.js

```diff
@@ -320,46 +346,48 @@ function CategoriesDetailCtrl($scope, $http){
       color: '#003057', //color - optional: choose your own line color.
       strokeWidth: 2,
     })
-  });
-
-  // line graph
-  $scope.categoryLine = {
-    chart: {
-      type: 'lineChart',
-      height: 450,
-      margin : {
-        top: 20,
-        right: 20,
-        bottom: 40,
-        left: 55
-      },
-      x: function(d){ return d.x; },
-      y: function(d){ return d.y; },
-      useInteractiveGuideline: true,
-      xScale: d3.time.scale(),
-      xAxis: {
-        axisLabel: 'Date',
-        tickFormat: function(d) {
-                        return d3.time.format('%m/%d/%y')(d)
-                    }
-      },
-      yAxis: {
-        axisLabel: 'Hours',
-        tickFormat: function(d){
-          return d3.format('.02f')(d);
+    // line graph
+    $scope.categoryLine = {
```

Comment:
Will wait to do this. Actually i think Cindy will do this in her PR because she refactored a bunch of line graph logic and I think she binded to controller. @cindytung JS god